### ISSUE TITLE
Iguana Spawner not spawning

### ISSUE THREAD

**Issue by Immaculala** [Tropicraft/Tropicraft#252]
I found a 'Dungeon Tree'. It has a couple of Iguana spawners in it, but is not spawning them.
It has the usual animation of a spawner, but just ain't spawning :(
No, I am not in Peaceful mode lol.
I am only using a few additional mods (JEI, Journeymap  etc.) but it is not a conflict with any of them because I removed everything except Tropicraft and it still didn’t spawn.
tropicraft-MC1.12.2-7.1.2.57
coroutil-1.12.1-1.1.24

**Comment by Winddbourne** [comment]
I'm having this issue too . . . the ashen aren't spawning at the top of a tree, and the mob spawners in the roguelike dungeons also aren't spawning mobs. I was just going to try using recurrent complex to move one out of the dimension to see if that fixed the problem but I searched here first.

**Comment by Winddbourne** [comment]
Since you just moved it to "important"  I did the test . . . moving the spawner to several dimensions. It worked fine in the Erebus and the nether but wouldn't spawn in the Overworld, Tropiccraft, or the twilight forest.  The special spawners in the hollow hill DID work in the twilight forest though. 

My pack has 205 mods so tracking down a conflict could be tedious I'll have to double check my results later.

**Comment by Winddbourne** [comment]
I did some more testing this morning since I was looking at the problem on no sleep at 5AM . . . I think the problem with the other dimensions was a lighting error on my part due to the light level in the twilight forest and my using the end sky in the overworld.  Both of those dimensions can look very dark and still have a light level of 15. 

I was able to get the spawner to work in those dimensions by changing the time and eliminating all outside light sources.  Which means the only world where the spider spawner doesn't work in the dark is the tropical one; plus I'm fairly sure the Ashen are supposed to spawn in daylight.

Yes the spawner still doesn't work in the tropical world even at night.

I shouldn't do these tests when I'm half asleep no matter how motivated I am to get to the bottom of an issue . . . such testing can result in me being an error. ROFL
